README.md

```diff
@@ -53,8 +53,8 @@ There is a manifest to deploy DVO via OLM artifacts.  This assumes that OLM is a
 
 ```
 # deploy this if you DO NOT want OLM to automatically upgrade DVO
-
-oc process --local NAMESPACE_IGNORE_PATTERN='openshift.*|kube-.+' -f deploy/openshift/deployment-validation-operator-olm.yaml | oc create -f -
+oc new-project deployment-validation-operator
+oc process --local NAMESPACE_IGNORE_PATTERN='openshift.*|kube-.+' -f deploy/openshift/deployment-validation-operator-olm.yaml | oc apply -f -
```

Comment:
Now that we have a better pattern ignore do we want to include this instead? NAMESPACE_IGNORE_PATTERN='^(openshift.*|kube-.*|default|dedicated-admin|redhat-.*|acm|addon-dba-operator|codeready-.*|prow)$' -f ...

Reply:
Uh, I think we should leave out the OSD stuff, but I will add a note about it.